lesson1/src/arrays/arrayProblems.js

```diff
@@ -9,7 +9,8 @@ class ArrayProblems {
   static allEqualElements(array) {
     for (let i = 0; i < array.length; i++) {
       for (let j = i; j < array.length; j++) {
-        // todo: do stuff here
+        // todo: do stuff herecd
+        if( array[i] !== array[j] ) return false;
```

Comment:
👏 nice job ... but whats the order of this algorithm ... and think (no need to do it ... if you could do better)